CONTRIBUTING.md

```diff
@@ -22,17 +22,42 @@ use GitHub Pull Requests (PR) for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using PRs.
 
+For a new review, the author works on a new branch on their own fork of the
+repository (we don't create additional branches on the main repository).
+
+Once the changes are made locally, the author creates a new commit on their
+newly created branch based on master, then pushes to their own fork, and creates
+a PR from the GitHub UI. The author then selects one or more reviewers.
```

Comment:
Might be a bit tricky for an external contributor to select a reviewer?

Reply:
Yeah, hopefully some of us will notice though.